kornia/utils/image.py

```diff
@@ -141,3 +142,18 @@ def tensor_to_image(tensor: torch.Tensor) -> np.array:
             "Cannot process tensor with shape {}".format(input_shape))
 
     return image
+
+
+class ImageToTensor(nn.Module):
+    """Converts a numpy image to a PyTorch 4d tensor image.
```

Comment:
By numpy image, are we assuming it is uint8 or floating points within 0~1?

Reply:
both - it just permutes, we don;t modify dtype

Reply:
That's the problem. If I am a user, I would expect this function returns a value as a Kornia image, which is shaped as BCHW and ranged from 0-1.

Reply:
probably we could provide an extra operator `TensorToDtype` or similar.

Reply:
@shijianjian another option is the following:
```python
def image_to_tensor(image: np.ndarray, keepdim: bool = True, device: Optional[torch.device] = None, dtype: Optional[torch.dtype] = None) -> torch.Tensor:
    ...
    tensor = ...
    if dtype is None:
        dtype = tensor.dtype
    return tensor.to(device, dtype)
```

Reply:
What I meant is the conversion from 8bit image, 12bit image, or 16bit image to 0-1 scale.

Reply:
for that we have normalize. With the definition below you could do the job
```python
nn.Sequential(
  ImageToTensor(dtype=torch.float32),
  Normalise(mean=0., std=255.),
)
```
I wouldn't change the default behavior of `image_to_tensor` since it might a large set of users